listenbrainz/webserver/static/js/src/stats/UserArtistMap.tsx

```diff
@@ -196,7 +197,7 @@ export default class UserArtistMap extends React.Component<
                       this.changeSelectedMetric("listen", event)
                     }
                   >
-                    Listens
+                    number of listens
```

Comment:
I'm not sure about this change because a normal user wouldn't care too much about the long wordings here. For me just Listens or Artists was more clear. Also, the sentence can be capitalized (the first alphabet) if we were to continue with this.